doc/conf.py

```diff
@@ -22,9 +20,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = setup_args['version']
+version = "1.7"
 # The full version, including alpha/beta/rc tags.
-release = setup_args['version']
+release = "1.7dev7"
```

Comment:
What is wrong with ``setup_args['version']``? My new process for making dev releases included keeping ``setup.py`` up-to-date. This approach will require updating on every dev-release...